federated/model-registry-ui-deployment.yaml

```
@@ -0,0 +1,7 @@
+- op: add
+  path: /spec/template/spec/containers/0/args
+  value: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)